src/hooks/useHandleWindowMessage.js

```diff
@@ -8,6 +8,7 @@ const WindowMessageSources = {
 const WindowMessageTypes = {
   URL_CHANGED: 'URL_CHANGED',
   JOB_ADDED: 'JOB_ADDED',
+  HTTP_STATUS_CODE: 'HTTP_STATUS_CODE',
```

Comment:
Maybe we should rename it to HTTP_ERROR_CODE since it only transmits those right?